inc/render/class-posts-grid-block.php

```diff
@@ -360,7 +360,7 @@ protected function retrieve_posts( $attributes, $count_posts, $page_number, &$to
 			'post_type'        => $attributes['postTypes'],
 			'posts_per_page'   => $attributes['postsToShow'],
 			'post_status'      => 'publish',
-			'order'            => $attributes['order'],
+			'order'            => $attributes['order'] ?? 'DESC',
```

Comment:
Note: We mention in the readme that we are compatible with 5.6, but `??` is a PHP 7.X feature.